doc/source/reference/commands/logical_count.rst

```diff
@@ -564,7 +564,50 @@ details.
 
 .. note::
 
-   Window function over multiple tables aren't supported yet.
+   Window function over multiple tables aren't supported version 9.0.1 or before.
+   Groonga supported it version 9.0.2 or later.
+   However, we need to align the same order for shard key and leading group key or sort key.
+
```

Comment:
It looks good, but how about describe a bit more? (the following one is just an idea. may be improved.)

```
... the order of target records (which is specified by leading group_keys or sort_keys ) should be aligned.

The order of target records should be the same in the following case.

* Specified by shard key and leading group key ( --group_keys LEADING_GROUP_KEY,... )
* Specified by shard key and leading sort key ( --sort_keys LEADING_SORT_KEYS,... )